cli/cmd/sudo/project/search.go

```diff
@@ -50,6 +57,89 @@ func SearchCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
+			if status {
+				var table []*projectStatusTableRow
+				ch.Printer.Println()
+				for _, name := range res.Names {
+					org := strings.Split(name, "/")[0]
+					project := strings.Split(name, "/")[1]
+
+					proj, err := client.GetProject(ctx, &adminv1.GetProjectRequest{
+						OrganizationName: org,
+						Name:             project,
+					})
+					if err != nil {
+						return err
+					}
+
+					depl := proj.ProdDeployment
+					if depl == nil {
+						continue
+					}
+
+					rt, err := runtimeclient.New(depl.RuntimeHost, proj.Jwt)
+					if err != nil {
+						return fmt.Errorf("failed to connect to runtime: %w", err)
+					}
+
+					res, err := rt.ListResources(cmd.Context(), &runtimev1.ListResourcesRequest{InstanceId: depl.RuntimeInstanceId})
+					if err != nil {
+						return fmt.Errorf("failed to list resources: %w", err)
+					}
+
+					var parser *runtimev1.ProjectParser
+					var ParserErrorCount int32
+					var IdleCount int32
+					var IdleWithErrorsCount int32
+					var PendingCount int32
+					var RunningCount int32
```

Comment:
1. Not external variables, so should start with lowercase
2. Can just use `int` (instead of `int32`)